src/js/quilt/redux/middleware/analytics.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Analytic middleware.
+ *
+ * This middleware is currently tied to Quilt project.
+ */
+
+import ReactGA from 'react-ga';
+import TrackingSchema from '../../../wellcome/TrackingSchema';
+import {JOURNEY_START, JOURNEY_EXIT, JOURNEY_NEXT, JOURNEY_PREV} from '../actions/journeyActions';
+import {WORLD_CLICK} from '../../../redux/actions/viewerActions';
+
+class Store {
+  static isInit = false;
+  static prevAction = null;
+  static prevId = null;
+  static prevIndex = null;
+}
+
+export const ga = (event) => {
+  if (process.env.__DEV__) {
+    if (!event) {
```

Comment:
Mostly for local debugging! Should never happen, but it was intended to catch this: https://github.com/wellcometrust/annotation-viewer/pull/12/files/266011ab5a84c283bd5a73c21b0bf02470f3c8db#diff-f115e47f8e6b6339ae4d67296b7f3f6aR76